api-report/driver-base.api.md

```diff
@@ -19,6 +19,7 @@ import { ISignalClient } from '@fluidframework/protocol-definitions';
 import { ISignalMessage } from '@fluidframework/protocol-definitions';
 import { ITelemetryLogger } from '@fluidframework/common-definitions';
 import { ITokenClaims } from '@fluidframework/protocol-definitions';
+import { MonitoringContext } from '@fluidframework/telemetry-utils';
```

Comment:
@anthony-murphy, @andre4i - Alternate naming idea is **DevopsContext**.  Basically, stuff that is not part of the core business logic of Fluid, but necessary to leverage it in an app/service at scale.

MonitoringContext is fine too, it hints at the same thing, but I heard Tony wonder if there was a better name that was a more natural home for flighting/config, so wanted to share in case it resonated.

Reply:
I don't like Devops context, but i'm having trouble articulating why, so it could be a good option 😁Maybe you can help assuage my nebulous reservations.

I think the first is around messaging. Does devops as a term bring with it any connotations that we may not want? will it scare developers away thinking they need to follow devops, or know something about operations to use the fluid framework? will people confuse this with azure dev ops since both that and fluid framework are from Microsoft?

The second is the term itself. To me devops is an adjective, and i usually name with nouns and verbs, this is a thing, or this does a thing. So i'm not really sure what devops here does to modify the term context, it feels buzzwordy to me, and not really descriptive. 

I fully admit the monitoring also only really covers half the usage well, the logging, so its also not perfect, but it feel more neutral to me.

Also, these don't feel like they exactly express my reservations, but like i said, i having trouble articulating it, so open to being convinced.

Reply:
MonitoringContext is fine, as we just discussed.  I get the uneasiness about implying more than it really means :)